### PR TITLE
Change valohai-utils learning path to be in sync with valohai/tensorf…

### PR DESCRIPTION
**Pull request by orasimus**
…low-example

I'm in the process of updating the `tensorflow-example` to use TensorFlow 2.6.0 and this adds parity between the code presented in the docs and the example.

There are two minor differences:
1) The `tensorflow-example` extracts normalizing the pixel values from 0-255 to 0.0-1.0 into a preprocessing step.

2) The `tensorflow-example` encapsulates all scripts into a 
```python
def main():
    ...

if __name__ == '__main__':
    main()
```

While following this specific best practice might be a bit of an overkill in the example, I feel it definitely would be in the docs.

### CODE REVIEW

File: source/tutorials/learning-paths/fundamentals/valohai-utils/3-outputs.rst [hunk 1]
```diff
@@ -3,54 +3,51 @@
...
 
-    mnist_file_path = 'mnist.npz'
+    valohai.prepare(
+        step='train-model',
+        image='tensorflow/tensorflow:2.6.0',
+    )
 
-    with numpy.load(mnist_file_path, allow_pickle=True) as f:
+    input_path = 'mnist.npz'
+    with numpy.load(input_path, allow_pickle=True) as f:
```
Comment (DrazenDodik) [inline source/tutorials/learning-paths/fundamentals/valohai-utils/3-outputs.rst:27]:
```suggestion
    with np.load(input_path, allow_pickle=True) as f:
```

File: source/tutorials/learning-paths/fundamentals/valohai-utils/6-metadata.rst [hunk 2]
```diff
@@ -2,71 +2,136 @@
...
+        default_inputs={
+            'dataset': 's3://onboard-sample/tf-sample/mnist.npz'
+        },
+        default_parameters={
+            'learning_rate': 0.001,
+            'epochs': 5,
+        },
+    )
+
+    input_path = valohai.inputs('mnist').path()
```
Comment (DrazenDodik) [inline source/tutorials/learning-paths/fundamentals/valohai-utils/6-metadata.rst:41]:
```suggestion
    input_path = valohai.inputs('dataset').path()
```
